packages/gestalt/src/Sheet.jsdom.test.js

```diff
@@ -296,4 +296,233 @@ describe('Sheet', () => {
 
     expect(mockOnDismiss).toHaveBeenCalledTimes(0);
   });
+
+  it('should not dismiss from the backdrop click when closeOnOutsideClick and dismissConfirmation are true', () => {
+    const mockOnDismiss = jest.fn();
+
+    render(
+      <Sheet
+        dismissConfirmation
+        accessibilityDismissButtonLabel="Dismiss"
+        accessibilitySheetLabel="Test Sheet"
+        closeOnOutsideClick
+        onDismiss={mockOnDismiss}
+      >
+        <section />
+      </Sheet>,
+    );
+    // eslint-disable-next-line testing-library/no-node-access -- Please fix the next time this file is touched!
+    const backDrop = screen.getByRole('dialog').parentElement?.firstElementChild;
+    if (!(backDrop instanceof HTMLElement)) {
+      throw new Error('Backdrop should be an HTMLElement');
```

Comment:
Throwing in a test?

Reply:
DONE

Reply:
I'm still seeing the line about throwing an error. Not a blocker, just seems odd to need to throw in a test?